File: Projet Python VS/Web 1/Dog.cs
using System;

public class Dog
{
	public string Name;
	public int Age;
	public List<string> Owners;
	public List<string> Tricks;


	public Dog(string Name,int Age,string Owners)
	{
		this.Name = Name;
		this.Age = Age;
		this.Owners = Owners;
		Tricks = new List<string>();
	}

	public void Learn(string trick)
    {
		this.Tricks.Add(trick);
    }
}

File: Projet Python VS/Projet Python VS/Dog.py

class Dog:
    def __init__(self,name,age,owners):
        self.name = name
        self.age = age
        self.owners = owners
        self.tricks = [] 

    def learn(self,trick):
        self.tricks.append(trick)

    def compare(self,dog):
        i = 0
        while i < len(self.tricks):
            if self.tricks[i] in dog.tricks:
                print(self.tricks[i])
            i+=1
        
